test/recycle_test.js

```diff
@@ -12,38 +13,6 @@ describe("Recycle", function(){
 
 	afterEach(mockFs.restore);
 
-	it("Creates an error message when there is an es6 syntax error", function(done){
```

Comment:
Nothing that this test is removed because it's no longer needed. The reason there was an error is because the modules would actually be transpiled by traceur/babel. But with [system-trace](https://github.com/stealjs/system-trace) you can prevent this with `System.preventModuleExecution = true` which I've added.